tests/test_testpath.py

```diff
@@ -137,10 +137,8 @@ def test_inference_git_submodule(self):
             self._run_command(
                 ['git', 'init',
                  str(temppath.joinpath("gitrepo"))])
-            self._run_command([
-                'git', 'submodule', 'add',
-                str(temppath.joinpath("submod")), 'submod'
-            ],
+            self._run_command(
+                ['git', '-c', 'protocol.file.allow=always', 'submodule', 'add', str(temppath.joinpath("submod")), 'submod'],
```

Comment:
This change is needed to workaround git-shell update happened 18th Oct, 2022.
https://bugs.launchpad.net/ubuntu/+source/git/+bug/1993586